parent-poms/pom.xml

```diff
@@ -816,7 +840,7 @@
                                         <exclude>org.codehaus.mojo:versions-maven-plugin:2.2</exclude>
                                     </excludes>
                                 </bannedDependencies>
-                                <dependencyConvergence/>
```

Comment:
Now I remember this was OK, it just too FOREVER to run 60 minutes+ so I removed while tinkering. Uncommenting this now.